src/clientlist.cpp

```diff
@@ -399,6 +399,16 @@ void unmanage_client(Window win) {
     if (m) monitor_apply_layout(m);
     ewmh_remove_client(win);
     tag_set_flags_dirty();
+
+    // Get the current client and update the windows focus.
+    client = frame_focused_client(tag->frame);
+    if (client) {
+        client_window_focus(client);
+    } else {
+        char buf[STRING_BUF_SIZE];
+        snprintf(buf, STRING_BUF_SIZE, "0x%x", (unsigned int) 0);
+        hook_emit_list("window_title_changed", buf, "", NULL);
```

Comment:
The window with the ID 0 is the root window - I think it'd be better to emit `window_title_changed` with no arguments at all, i.e. `hook_emit_list("window_title_changed", NULL);` if I'm not mistaken.